### PR TITLE
[INLONG-6289][Docker][Audit] Add configuration to choose starting proxy or store or both 

### LINKED ISSUE #6289: [Feature][Docker][Audit] Add configuration to choose starting proxy or store or both
### Description

Add configuration to choose starting proxy or store or both

### Use case

Add configuration to choose starting proxy or store or both

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by lucaspeng12138**
### Prepare a Pull Request

[INLONG-6289][Docker][Audit] Add configuration to choose starting proxy or store or both 
- Fixes #6289 

### Motivation

Add configuration to choose starting proxy or store or both 

### Modifications

Add configuration to choose starting proxy or store or both 

### Verifying this change

Add configuration to choose starting proxy or store or both

### CODE REVIEW

File: inlong-audit/audit-docker/Dockerfile [hunk 1]
```diff
@@ -31,6 +31,7 @@ ENV PULSAR_AUDIT_TOPIC="persistent:\/\/public\/default\/inlong-audit"
 ENV TUBE_MASTER_LIST="localhost:8715"
 ENV TUBE_AUDIT_TOPIC="inlong-audit"
 ENV AUDIT_DBNAME="apache_inlong_audit"
+ENV START_MODE="both"
```
Comment (dockerzhang) [inline inlong-audit/audit-docker/Dockerfile:34]: START_MODE="both"
->
START_MODE="all"

Comment (lucaspeng12138) [reply inlong-audit/audit-docker/Dockerfile]: fixed

File: inlong-audit/audit-docker/Dockerfile [hunk 2]
```diff
@@ -31,6 +31,7 @@ ENV PULSAR_AUDIT_TOPIC="persistent:\/\/public\/default\/inlong-audit"
 ENV TUBE_MASTER_LIST="localhost:8715"
 ENV TUBE_AUDIT_TOPIC="inlong-audit"
 ENV AUDIT_DBNAME="apache_inlong_audit"
+ENV START_MODE="all"
```
Comment (dockerzhang) [inline inlong-audit/audit-docker/Dockerfile:34]:
```suggestion'
# proxy/store/all, start audit module individually, or all
ENV START_MODE="all"
```

Comment (lucaspeng12138) [reply inlong-audit/audit-docker/Dockerfile:35]: fixed

**Review by dockerzhang — APPROVED** [review]

**Review by healchow — APPROVED** [review]

**Pull request merged**